packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -5,6 +5,7 @@ export type HintDisplayText = {
   hintCanNotIdentifyText?: string;
   hintTooCloseText?: string;
   hintTooFarText?: string;
+  /** @deprecated String is no longer in use */
```

Comment:
What do you think of adding some communication regarding removal here?
```suggestion
 /** @deprecated `hintHoldFacePositionCountdownText` is no longer in use and will be removed in a future major version release. */
```

Reply:
I'm on board with it